### ISSUE TITLE
React import issue

### ISSUE THREAD

**Issue by vimal1998** [jonataswalker/ol-contextmenu#211]
@jonataswalker  

Could not import the contextmenu in react
```
"Could not find a declaration file for module 'ol-contextmenu'. '/node_modules/ol-contextmenu/dist/ol-contextmenu.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/ol-contextmenu` if it exists or add a new declaration (.d.ts) file containing `declare module 'ol-contextmenu';`"
```

**Comment by tarangrathod** [comment]
did you tried like this 
import ContextMenu from "ol-contextmenu";

it worked for me
here is my example [url ](https://codesandbox.io/embed/wmts-dimensions-forked-sw1uqd?fontsize=14&hidenavigation=1&theme=dark)

**Comment by jonataswalker** [comment]
Hi @tarangrathod 

I've just published `5.0.0-alpha`. Are you able to help with a validation?

PR still opened
https://github.com/jonataswalker/ol-contextmenu/pull/219

**Comment by jonataswalker** [comment]
Should be solved with #219
